scripts/presubmit-tests.sh

```diff
@@ -211,19 +211,19 @@ function run_unit_tests() {
   local failed=0
   # Run pre-unit tests, if any
   if function_exists pre_unit_tests; then
-    pre_unit_tests || failed=1
+    pre_unit_tests || { failed=1; step_failed "pre_unit_tests"; }
```

Comment:
It seems we do not need this `;` in the end either?

Reply:
Hmm, I saw your comment above, so this ; in the end is needed?

Reply:
yep it's needed